src/Propel/Generator/Util/QuickBuilder.php

```diff
@@ -114,17 +127,17 @@ public function getSchemaName()
      *
      * @return void
      */
-    public function setParser($parser)
+    public function setParser(SchemaParserInterface $parser): void
     {
         $this->parser = $parser;
     }
 
     /**
-     * @return \Propel\Generator\Reverse\SchemaParserInterface
+     * @return \Propel\Generator\Reverse\SchemaParserInterface|null
      */
-    public function getParser()
+    public function getParser(): ?SchemaParserInterface
     {
-        return $this->parser;
+        return $this->parser ?? null;
```

Comment:
Isn't `$this->parser` `null` by default?

Reply:
The docblock in property needs to be fixed then to append `|null`, as well

Reply:
Nope. Starting from PHP 7.4, when you use strict types, the properties aren't initialize to null anymore. If `$this->parser` is not set, it's `uninitialized` and can't be accessed.

Reply:
I adjusted the code to change it as little as possible.